CDS/WebContent/WEB-INF/jsps/layout/head.jsp

```diff
@@ -145,7 +145,7 @@
 
               <ul class="nav nav-treeview">
                 <% for (int i = 0; i < menuPages.length; i++) 
-                   if ((i > 0 && i < 5) || Role.isAdmin(request)) { %>
+                   if ((i > 0 && i < 6) || Role.isAdmin(request)) { %>
```

Comment:
Maybe at some point we should do this differently than hardcode the number of team menu items. Nor for now

Reply:
Yes :)